Classes/IndexQueue/RecordMonitor.php

```diff
@@ -98,7 +98,7 @@ public function processCmdmap_postProcess(
 
         // track publish / swap events for records (workspace support)
         // command "version"
-        if ($command === 'version' && $value['action'] === 'swap') {
+        if ($command === 'version' && $value['action'] === 'publish') {
```

Comment:
```suggestion
        if ($command === 'version' && ($value['action'] === 'publish' || $value['action'] === 'swap')) {
```


The command map for 'swap' is still present on TYPO3 13.4.19, 
See mention on https://github.com/TYPO3-Solr/ext-solr/pull/3938#issuecomment-1960861595:

https://github.com/TYPO3/typo3/blob/v13.4.19/typo3/sysext/workspaces/Classes/Service/WorkspaceService.php#L167

What does it mean for EXT:solr exactly? Does it break some functionality of EXT:solr?
May be just leave 'swap' check there as well?